components/brave_shields/browser/ad_block_base_service.cc

```diff
@@ -271,6 +272,82 @@ void AdBlockBaseService::ResetForTest(const std::string& rules,
   AddKnownResourcesToAdBlockInstance();
 }
 
+
+// Merges the contents of the second HostnameCosmeticResources Value into the
+// first one provided.
+//
+// If `force_hide` is true, the contents of `from`'s `hide_selectors` field
+// will be moved into a possibly new field of `into` called
+// `force_hide_selectors`.
+void MergeResourcesInto(
```

Comment:
I'd move this function into `ad_block_service_helper`, that's where we've been putting "helper" functions like this.